analyzers/tests/SonarAnalyzer.UnitTest/MetadataReferences/WpfMetadataReference.cs

```diff
@@ -0,0 +1,33 @@
+﻿/*
+ * SonarAnalyzer for .NET
+ * Copyright (C) 2015-2022 SonarSource SA
+ * mailto: contact AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+#if NET
+
+using static SonarAnalyzer.UnitTest.MetadataReferences.MetadataReferenceFactory;
+
+namespace SonarAnalyzer.UnitTest.MetadataReferences
+{
+    internal static class WpfMetadataReference
+    {
+        internal static MetadataReference SystemXaml { get; } = Create(typeof(System.Xaml.XamlReader));
```

Comment:
This should go to `CoreMetadataReference.cs`

Reply:
We also have a dedicated class for Windows.Forms for this and in WPF you also have to distinguish between

* System.Xaml
* System.Printing
* System.IO.Packaging
* WindowsBase
* PresentationFramework
* PresentationCore
* and others

I think it makes sense to either separate all the frameworks (Core/WinFowms/WPF) or combine them all into one.

Reply:
Not relevant anymore.